.github/CODEOWNERS

```diff
@@ -1,2 +1,6 @@
 # These owners are the maintainers and approvers of this repo
-*       @radius-project/maintainers-docs @radius-project/approvers-docs
\ No newline at end of file
+*       @radius-project/maintainers-docs @radius-project/approvers-docs
+
+# Dependabot managed files
+docs/package.json        @radius-project/on-call
+docs/package-lock.json   @radius-project/on-call
```

Comment:
I have the same question on this PR as on the samples CODEOWNERS update: https://github.com/radius-project/samples/pull/2410#discussion_r2457587853

Reply:
Good catch! Updated